M/mtdev/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "mtdev"
+version = v"1.1.6"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://bitmath.org/code/mtdev/mtdev-$version.tar.gz", "1325f389a2f25cd5f5a8ea4d29aad24aa7c3ec30401d679400dd79eb9c0a8dbb"),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/mtdev-*/
+# build system doesn't recognize platforms ending with -musl
+_MACHTYPE=$(echo ${MACHTYPE} | sed 's/\(.*\)-musl$/\1/')
```

Comment:
Running [`update_configure_scripts`](https://docs.binarybuilder.org/stable/build_tips/#Utilities-in-the-build-environment) is probably a better option
```suggestion
update_configure_scripts
```

Reply:
Oh, nice! That's great to know!